code/HISPANIA/modules/clothing/under/jobs/security.dm

```diff
@@ -0,0 +1,26 @@
+///Hispania Civilians Clothes
+
+/*Nota: todos los sprites que sean pertenecientes al code hispania y tengan sus
+respectivos sprites en las carpetas de iconos de hispania , es decir icons/hispania
+deberan tener una linea de codigo demas para que funcionen "hispania_icon = TRUE"*/
+
+// Security Formal Uniform
+
+/obj/item/clothing/under/rank/security/private
+	name = "formal blue security officer's uniform"
+	desc = "A formal security officer's uniform , it have a logo says Security Department"
+	icon_state = "privateblue"
+	item_color = "privateblue"
+	hispania_icon = TRUE
+
+/obj/item/clothing/under/rank/security/private/red
+	name = "formal red security officer's uniform"
+	icon_state = "privatered"
+	item_color= "privatered"
+	hispania_icon = TRUE
+
+/obj/item/clothing/under/rank/security/pod_pilot/formal
+	name = "formal pod pilot's jumpsuit"
```

Comment:
Deberías agregar una desc a este item a su vez. Si hereda la desc únicamente dirá 
desc = "Suit for your regular pod pilot."